release/pkg/assets_eksd.go

```diff
@@ -68,7 +68,7 @@ func (r *ReleaseConfig) GetEksDChannelAssets(eksDReleaseChannel, kubeVer, eksDRe
 
 			if r.DevRelease || r.ReleaseEnvironment == "development" {
 				sourceS3Key = fmt.Sprintf("%s.%s", osName, imageExtensions[imageFormat])
-				sourceS3Prefix = fmt.Sprintf("%s/%s/%s/%s/%s", imageBuilderProjectPath, eksDReleaseChannel, imageFormat, osName, latestPath)
+				sourceS3Prefix = fmt.Sprintf("%s/%s/%s/%s/%s", imageBuilderProjectPath, imageFormat, eksDReleaseChannel, osName, latestPath)
```

Comment:
did we look to see if the dev release structure will break anything?

Reply:
https://tiny.amazon.com/v5ha8lwx/IsenLink

Reply:
it was wrong as well.

Reply:
Was taking a look but can't find the artifact listed under this: https://github.com/aws/eks-anywhere/blob/5fc329d3390156532eebc0ebfd29aa65cf8f4a63/release/pkg/assets_eksd.go#L71

Reply:
Let's not change this just because the imageBuilderProjectPath prefix isn't something we are able to find right now